google/resource_google_project_default_service_accounts.go

```diff
@@ -87,7 +94,13 @@ func resourceGoogleProjectDefaultServiceAccountsDoAction(d *schema.ResourceData,
 		}
 	case "ENABLE":
 		_, err := config.NewIamClient(userAgent).Projects.ServiceAccounts.Enable(serviceAccountSelfLink, &iam.EnableServiceAccountRequest{}).Do()
-		if err != nil {
+		errReceived := err != nil
+		errExpected := restorePolicy == "REVERT_AND_IGNORE_FAILURE"
+		if errExpected && errReceived {
+			log.Printf("cannot enable service account %s: %v", serviceAccountSelfLink, err)
```

Comment:
```suggestion
			log.Printf("[DEBUG]  cannot enable service account %s: %v", serviceAccountSelfLink, err)
```

We prefix log lines with what type of message it is

Reply:
The other log.Printf should have the same prefix as well

Reply:
done